### PR TITLE
Move distro image in Dashboard

### LINKED ISSUE #699: Put distro image first like datacenter image

### PR DESCRIPTION
**Pull request by na3d**
Closes #699 

Messed up my git trying to type too fast.

### CODE REVIEW

File: scss/manager.scss [hunk 1]
```diff
@@ -698,6 +698,10 @@ hr {
     opacity: 0.7;
 }
 
+.distro-style img {
```
Comment (eatonphil) [inline scss/manager.scss]: Again, I would just add this selector to the selector list in line 694 so you aren't making up your own margin-right here and above.

Comment (na3d) [inline scss/manager.scss]: They go to two different things, and need to function differently.

**Comment by na3d** [comment]
screen shot 2016-11-18 at 9 52 55 am

File: scss/manager.scss [hunk 2]
```diff
@@ -885,6 +885,10 @@ table {
         margin-left: 0.5rem;
     }
 
+    .distro-style img {
```
Comment (eatonphil) [inline scss/manager.scss]: Fine, but no need to restrict this to being inside a content-col.

**Pull request merged**